src/js/services/forms.js

```diff
@@ -156,7 +156,7 @@ export default App.extend({
       _patient: this.patient.getResource(),
     }, { limit, sort })
       .then(field => {
-        this.send(message, { value: field.get('value') }, requestId);
+        this.send(message, { value: field.get('values') }, requestId);
```

Comment:
@zfixler this would be the reason you were getting `undefined` results when using the history API, the code was trying to pull a non-existent key.

Reply:
well all keys were non-existent since a model can't parse an array